Components/BlockImageTextParallax/script.js

```diff
@@ -0,0 +1,51 @@
+import Rellax from 'rellax'
+import $ from 'jquery'
+
+const $window = $(window)
+
+let rellax, initialized
```

Comment:
These should be part of the component's state instead, please see best practices.